next-tavla/pages/boards/index.tsx

```diff
@@ -23,28 +26,32 @@ export async function getServerSideProps({
         }
     }
     const boards = await getBoardsForUser(user.uid)
+    const organizations = await getOrganizationsWithUser(user.uid)
 
     return {
         props: {
             boards: boards,
+            organizations: organizations,
             loggedIn: user !== null,
         },
     }
 }
 
 function OverviewPage({
     boards,
+    organizations,
     loggedIn,
 }: {
     boards: TBoard[]
+    organizations: TOrganization[]
     loggedIn: boolean
 }) {
     return (
         <div className={classes.root}>
             <AdminHeader loggedIn={loggedIn} />
```

Comment:
Samme som over, ser at det er jeg som skrev den for noen uker siden men grei å ta i samme slengen 😂